src/compiler/transformers/test/parse-form-associated.spec.ts

```diff
@@ -10,10 +10,10 @@ describe('parse form associated', function () {
     export class CmpA {
     }
     `);
-    expect(t.cmp.formAssociated).toBe(true);
+    expect(t.cmp!.formAssociated).toBe(true);
```

Comment:
I'm guessing based on the commit msg " if those aren't there we should get a runtime failure in the tests anyway" these were causing SNC errors?

Reply:
yup that's right, and since it's a test I'm not bothered at all by saying "go ahead and access this"

Reply:
Gotcha, totally makes sense in this context - thanks!